### PR TITLE
Add a litmusbook for cstor_pool_scale

### PR DESCRIPTION
**Pull request by shashank855**
**What this PR does / why we need it**:
- Adds a litmusbook for cstor_pool_scale.
- *Idea behind the test* is to change the deploymnet spec of any one of the pool pod and making replica-count as 0 so that one of the pool pod goes in terminating state.
- Then perform some I/Os this will cause inconsistency between CVRs(i.e:Allocated_Space).
- Then bring the terminated pool pod again check whether the pool pods are in sync and the CVRs are consistant or not. 

**Following is the log of the ansibletest container**:
```
PLAYBOOK: test.yml *************************************************************
1 plays in ./percona/chaos/cstor_pool_scale/test.yml

PLAY [localhost] ***************************************************************

TASK [Gathering Facts] *********************************************************
task path: /percona/chaos/cstor_pool_scale/test.yml:2
ok: [127.0.0.1]
META: ran handlers

TASK [Record test instance/run ID] *********************************************
task path: /percona/chaos/cstor_pool_scale/test.yml:15
skipping: [127.0.0.1] => {"changed": false, "skip_reason": "Conditional result was False"}

TASK [Construct testname appended with runID] **********************************
task path: /percona/chaos/cstor_pool_scale/test.yml:19
skipping: [127.0.0.1] => {"changed": false, "skip_reason": "Conditional result was False"}

TASK [include_tasks] ***********************************************************
task path: /percona/chaos/cstor_pool_scale/test.yml:25
included: /common/utils/update_litmus_result_resource.yml for 127.0.0.1

TASK [Generate the litmus result CR to reflect SOT (Start of Test)] ************
task path: /common/utils/update_litmus_result_resource.yml:3
changed: [127.0.0.1] => {"changed": true, "checksum": "918ae039271c213632c2f68120faddc18feac4cc", "dest": "./litmus-result.yaml", "gid": 0, "group": "root", "md5sum": "bfa3431ef59db91c95f31ea1e148dcd5", "mode": "0644", "owner": "root", "size": 433, "src": "/root/.ansible/tmp/ansible-tmp-1545300456.46-101665192007181/source", "state": "file", "uid": 0}

TASK [Analyze the cr yaml] *****************************************************
task path: /common/utils/update_litmus_result_resource.yml:14
changed: [127.0.0.1] => {"changed": true, "cmd": "cat litmus-result.yaml", "delta": "0:00:00.299566", "end": "2018-12-20 10:07:37.356523", "rc": 0, "start": "2018-12-20 10:07:37.056957", "stderr": "", "stderr_lines": [], "stdout": "---\napiVersion: litmus.io/v1alpha1\nkind: LitmusResult\nmetadata:\n\n  # name of the litmus testcase\n  name: cstor-pool-scale \nspec:\n\n  # holds information on the testcase\n  testMetadata:\n    app:  \n    chaostype: cstor-pool-scale \n\n  # holds the state of testcase,  manually updated by json merge patch\n  # result is the useful value today, but anticipate phase use in future \n  testStatus: \n    phase: in-progress  \n    result: none ", "stdout_lines": ["---", "apiVersion: litmus.io/v1alpha1", "kind: LitmusResult", "metadata:", "", "  # name of the litmus testcase", "  name: cstor-pool-scale ", "spec:", "", "  # holds information on the testcase", "  testMetadata:", "    app:  ", "    chaostype: cstor-pool-scale ", "", "  # holds the state of testcase,  manually updated by json merge patch", "  # result is the useful value today, but anticipate phase use in future ", "  testStatus: ", "    phase: in-progress  ", "    result: none "]}

TASK [Apply the litmus result CR] **********************************************
task path: /common/utils/update_litmus_result_resource.yml:17
changed: [127.0.0.1] => {"changed": true, "cmd": "kubectl apply -f litmus-result.yaml", "delta": "0:00:00.960390", "end": "2018-12-20 10:07:38.473211", "failed_when_result": false, "rc": 0, "start": "2018-12-20 10:07:37.512821", "stderr": "", "stderr_lines": [], "stdout": "litmusresult.litmus.io/cstor-pool-scale configured", "stdout_lines": ["litmusresult.litmus.io/cstor-pool-scale configured"]}

TASK [Generate the litmus result CR to reflect EOT (End of Test)] **************
task path: /common/utils/update_litmus_result_resource.yml:27
skipping: [127.0.0.1] => {"changed": false, "skip_reason": "Conditional result was False"}

TASK [Analyze the cr yaml] *****************************************************
task path: /common/utils/update_litmus_result_resource.yml:38
skipping: [127.0.0.1] => {"changed": false, "skip_reason": "Conditional result was False"}

TASK [Apply the litmus result CR] **********************************************
task path: /common/utils/update_litmus_result_resource.yml:41
skipping: [127.0.0.1] => {"changed": false, "skip_reason": "Conditional result was False"}

TASK [Display the app information passed via the test job] *********************
task path: /percona/chaos/cstor_pool_scale/test.yml:32
ok: [127.0.0.1] => {
    "msg": [
        "The application info is as follows:", 
        "Namespace    : app-percona-ns", 
        "Label        : name=percona", 
        "PVC          : percona-mysql-claim"
    ]
}

TASK [Verify that the AUT (Application Under Test) is running] *****************
task path: /percona/chaos/cstor_pool_scale/test.yml:42
changed: [127.0.0.1] => {"attempts": 1, "changed": true, "cmd": "kubectl get pods -n app-percona-ns -l name=percona --no-headers -o custom-columns=:status.phase", "delta": "0:00:00.393832", "end": "2018-12-20 10:07:39.188164", "rc": 0, "start": "2018-12-20 10:07:38.794332", "stderr": "", "stderr_lines": [], "stdout": "Running", "stdout_lines": ["Running"]}

TASK [Get application pod name] ************************************************
task path: /percona/chaos/cstor_pool_scale/test.yml:53
changed: [127.0.0.1] => {"changed": true, "cmd": "kubectl get pods -n app-percona-ns -l name=percona --no-headers -o=custom-columns=NAME:\".metadata.name\"", "delta": "0:00:00.384297", "end": "2018-12-20 10:07:39.732560", "rc": 0, "start": "2018-12-20 10:07:39.348263", "stderr": "", "stderr_lines": [], "stdout": "percona-cb697f8b4-n4dgb", "stdout_lines": ["percona-cb697f8b4-n4dgb"]}

TASK [Getting the number of cstor-pool pods] ***********************************
task path: /percona/chaos/cstor_pool_scale/test.yml:61
changed: [127.0.0.1] => {"changed": true, "cmd": "kubectl get pods -n openebs -l app=cstor-pool --no-headers | wc -l", "delta": "0:00:00.383512", "end": "2018-12-20 10:07:40.265198", "failed_when_result": false, "rc": 0, "start": "2018-12-20 10:07:39.881686", "stderr": "", "stderr_lines": [], "stdout": "3", "stdout_lines": ["3"]}

TASK [Getting the volume name] *************************************************
task path: /percona/chaos/cstor_pool_scale/test.yml:66
changed: [127.0.0.1] => {"changed": true, "cmd": "kubectl get pvc -n app-percona-ns -o jsonpath='{.items[0].spec.volumeName}'", "delta": "0:00:00.382325", "end": "2018-12-20 10:07:40.803245", "rc": 0, "start": "2018-12-20 10:07:40.420920", "stderr": "", "stderr_lines": [], "stdout": "pvc-aedf9cbb-0434-11e9-9649-0ac72a660b58", "stdout_lines": ["pvc-aedf9cbb-0434-11e9-9649-0ac72a660b58"]}

TASK [Get CVR status list from pv] *********************************************
task path: /percona/chaos/cstor_pool_scale/test.yml:70
changed: [127.0.0.1] => {"changed": true, "cmd": "kubectl get cvr -n openebs -l openebs.io/persistent-volume=pvc-aedf9cbb-0434-11e9-9649-0ac72a660b58 --no-headers -o jsonpath='{range .items[*]}{.status.phase}{\"\\n\"}{end}'", "delta": "0:00:00.390599", "end": "2018-12-20 10:07:41.355896", "rc": 0, "start": "2018-12-20 10:07:40.965297", "stderr": "", "stderr_lines": [], "stdout": "Healthy\nHealthy\nHealthy", "stdout_lines": ["Healthy", "Healthy", "Healthy"]}

TASK [Check status of cvr] *****************************************************
task path: /percona/chaos/cstor_pool_scale/test.yml:79
changed: [127.0.0.1] => (item=Healthy) => {"changed": true, "cmd": ["echo", "Healthy"], "delta": "0:00:00.300978", "end": "2018-12-20 10:07:41.815285", "failed_when_result": false, "item": "Healthy", "rc": 0, "start": "2018-12-20 10:07:41.514307", "stderr": "", "stderr_lines": [], "stdout": "Healthy", "stdout_lines": ["Healthy"]}
changed: [127.0.0.1] => (item=Healthy) => {"changed": true, "cmd": ["echo", "Healthy"], "delta": "0:00:00.301399", "end": "2018-12-20 10:07:42.218751", "failed_when_result": false, "item": "Healthy", "rc": 0, "start": "2018-12-20 10:07:41.917352", "stderr": "", "stderr_lines": [], "stdout": "Healthy", "stdout_lines": ["Healthy"]}
changed: [127.0.0.1] => (item=Healthy) => {"changed": true, "cmd": ["echo", "Healthy"], "delta": "0:00:00.299380", "end": "2018-12-20 10:07:42.619078", "failed_when_result": false, "item": "Healthy", "rc": 0, "start": "2018-12-20 10:07:42.319698", "stderr": "", "stderr_lines": [], "stdout": "Healthy", "stdout_lines": ["Healthy"]}

TASK [Getting the pool deployment name] ****************************************
task path: /percona/chaos/cstor_pool_scale/test.yml:85
changed: [127.0.0.1] => {"changed": true, "cmd": "kubectl get deployment -l app=cstor-pool -n openebs --no-headers -o jsonpath='{.items[0].metadata.name}'", "delta": "0:00:00.386868", "end": "2018-12-20 10:07:43.160438", "rc": 0, "start": "2018-12-20 10:07:42.773570", "stderr": "", "stderr_lines": [], "stdout": "cstor-sparse-pool-4csk", "stdout_lines": ["cstor-sparse-pool-4csk"]}

TASK [Getting the cvr for the deployment] **************************************
task path: /percona/chaos/cstor_pool_scale/test.yml:89
changed: [127.0.0.1] => {"changed": true, "cmd": "kubectl get cvr -l cstorpool.openebs.io/name=cstor-sparse-pool-4csk -n openebs -o jsonpath='{.items[0].metadata.name}'", "delta": "0:00:00.380155", "end": "2018-12-20 10:07:43.692063", "rc": 0, "start": "2018-12-20 10:07:43.311908", "stderr": "", "stderr_lines": [], "stdout": "pvc-aedf9cbb-0434-11e9-9649-0ac72a660b58-cstor-sparse-pool-4csk", "stdout_lines": ["pvc-aedf9cbb-0434-11e9-9649-0ac72a660b58-cstor-sparse-pool-4csk"]}

TASK [Scaling-down cstor-pool] *************************************************
task path: /percona/chaos/cstor_pool_scale/test.yml:93
changed: [127.0.0.1] => {"changed": true, "cmd": "kubectl patch deployment cstor-sparse-pool-4csk -n openebs --type=json -p='[{\"op\":\"add\", \"path\":\"/spec/replicas\", \"value\":\"0\"}]'", "delta": "0:00:00.388426", "end": "2018-12-20 10:07:44.229000", "failed_when_result": false, "rc": 0, "start": "2018-12-20 10:07:43.840574", "stderr": "", "stderr_lines": [], "stdout": "deployment.extensions/cstor-sparse-pool-4csk patched", "stdout_lines": ["deployment.extensions/cstor-sparse-pool-4csk patched"]}

TASK [Checking for pool scale-down] ********************************************
task path: /percona/chaos/cstor_pool_scale/test.yml:98
FAILED - RETRYING: Checking for pool scale-down (5 retries left).
FAILED - RETRYING: Checking for pool scale-down (4 retries left).
FAILED - RETRYING: Checking for pool scale-down (3 retries left).
changed: [127.0.0.1] => {"attempts": 4, "changed": true, "cmd": "kubectl get pods -n openebs -l app=cstor-pool --no-headers | wc -l", "delta": "0:00:00.380360", "end": "2018-12-20 10:08:31.239004", "rc": 0, "start": "2018-12-20 10:08:30.858644", "stderr": "", "stderr_lines": [], "stdout": "2", "stdout_lines": ["2"]}

TASK [Create some test data in the mysql database] *****************************
task path: /percona/chaos/cstor_pool_scale/test.yml:105
included: /common/utils/mysql_data_persistence.yml for 127.0.0.1

TASK [Create some test data in the mysql database] *****************************
task path: /common/utils/mysql_data_persistence.yml:4
changed: [127.0.0.1] => (item=mysql -uroot -pk8sDem0 -e 'create database tdb;') => {"changed": true, "cmd": "kubectl exec percona-cb697f8b4-n4dgb -n app-percona-ns -- mysql -uroot -pk8sDem0 -e 'create database tdb;'", "delta": "0:00:00.459684", "end": "2018-12-20 10:08:31.940234", "failed_when_result": false, "item": "mysql -uroot -pk8sDem0 -e 'create database tdb;'", "rc": 0, "start": "2018-12-20 10:08:31.480550", "stderr": "mysql: [Warning] Using a password on the command line interface can be insecure.", "stderr_lines": ["mysql: [Warning] Using a password on the command line interface can be insecure."], "stdout": "", "stdout_lines": []}
changed: [127.0.0.1] => (item=mysql -uroot -pk8sDem0 -e 'create table ttbl (Data VARCHAR(20));' tdb) => {"changed": true, "cmd": "kubectl exec percona-cb697f8b4-n4dgb -n app-percona-ns -- mysql -uroot -pk8sDem0 -e 'create table ttbl (Data VARCHAR(20));' tdb", "delta": "0:00:00.527666", "end": "2018-12-20 10:08:32.576796", "failed_when_result": false, "item": "mysql -uroot -pk8sDem0 -e 'create table ttbl (Data VARCHAR(20));' tdb", "rc": 0, "start": "2018-12-20 10:08:32.049130", "stderr": "mysql: [Warning] Using a password on the command line interface can be insecure.", "stderr_lines": ["mysql: [Warning] Using a password on the command line interface can be insecure."], "stdout": "", "stdout_lines": []}
changed: [127.0.0.1] => (item=mysql -uroot -pk8sDem0 -e 'insert into ttbl (Data) VALUES ("tdata");' tdb) => {"changed": true, "cmd": "kubectl exec percona-cb697f8b4-n4dgb -n app-percona-ns -- mysql -uroot -pk8sDem0 -e 'insert into ttbl (Data) VALUES (\"tdata\");' tdb", "delta": "0:00:00.484982", "end": "2018-12-20 10:08:33.170575", "failed_when_result": false, "item": "mysql -uroot -pk8sDem0 -e 'insert into ttbl (Data) VALUES (\"tdata\");' tdb", "rc": 0, "start": "2018-12-20 10:08:32.685593", "stderr": "mysql: [Warning] Using a password on the command line interface can be insecure.", "stderr_lines": ["mysql: [Warning] Using a password on the command line interface can be insecure."], "stdout": "", "stdout_lines": []}

TASK [Checking for the Corrupted tables] ***************************************
task path: /common/utils/mysql_data_persistence.yml:21
skipping: [127.0.0.1] => {"changed": false, "skip_reason": "Conditional result was False"}

TASK [Verify mysql data persistence] *******************************************
task path: /common/utils/mysql_data_persistence.yml:30
skipping: [127.0.0.1] => {"changed": false, "skip_reason": "Conditional result was False"}

TASK [Delete/drop MySQL database] **********************************************
task path: /common/utils/mysql_data_persistence.yml:43
skipping: [127.0.0.1] => {"changed": false, "skip_reason": "Conditional result was False"}

TASK [Verify successful db delete] *********************************************
task path: /common/utils/mysql_data_persistence.yml:51
skipping: [127.0.0.1] => {"changed": false, "skip_reason": "Conditional result was False"}

TASK [Getting the cvr name of running pool] ************************************
task path: /percona/chaos/cstor_pool_scale/test.yml:115
changed: [127.0.0.1] => {"changed": true, "cmd": "kubectl get cvr -n openebs -o jsonpath='{.items[?(@.metadata.labels.cstorpool\\.openebs\\.io/name!=\"cstor-sparse-pool-4csk\")].metadata.name}' | awk '{print $1}'", "delta": "0:00:00.404415", "end": "2018-12-20 10:08:33.875579", "rc": 0, "start": "2018-12-20 10:08:33.471164", "stderr": "", "stderr_lines": [], "stdout": "pvc-aedf9cbb-0434-11e9-9649-0ac72a660b58-cstor-sparse-pool-6ect", "stdout_lines": ["pvc-aedf9cbb-0434-11e9-9649-0ac72a660b58-cstor-sparse-pool-6ect"]}

TASK [Scaling-up cstor-pool] ***************************************************
task path: /percona/chaos/cstor_pool_scale/test.yml:119
changed: [127.0.0.1] => {"changed": true, "cmd": "kubectl patch deployment cstor-sparse-pool-4csk -n openebs --type=json -p='[{\"op\":\"add\", \"path\":\"/spec/replicas\", \"value\":\"1\"}]'", "delta": "0:00:00.385597", "end": "2018-12-20 10:08:34.415507", "failed_when_result": false, "rc": 0, "start": "2018-12-20 10:08:34.029910", "stderr": "", "stderr_lines": [], "stdout": "deployment.extensions/cstor-sparse-pool-4csk patched", "stdout_lines": ["deployment.extensions/cstor-sparse-pool-4csk patched"]}

TASK [Checking for pool scale-up] **********************************************
task path: /percona/chaos/cstor_pool_scale/test.yml:124
FAILED - RETRYING: Checking for pool scale-up (5 retries left).
changed: [127.0.0.1] => {"attempts": 2, "changed": true, "cmd": "kubectl get pods -n openebs -l app=cstor-pool -o jsonpath='{.items[0].status.phase}'", "delta": "0:00:00.395071", "end": "2018-12-20 10:08:50.460085", "rc": 0, "start": "2018-12-20 10:08:50.065014", "stderr": "", "stderr_lines": [], "stdout": "Running", "stdout_lines": ["Running"]}

TASK [Getting the cvr status] **************************************************
task path: /percona/chaos/cstor_pool_scale/test.yml:131
changed: [127.0.0.1] => {"changed": true, "cmd": "kubectl get cvr pvc-aedf9cbb-0434-11e9-9649-0ac72a660b58-cstor-sparse-pool-4csk -n openebs -o jsonpath='{.status.phase}'", "delta": "0:00:00.382360", "end": "2018-12-20 10:08:50.997693", "rc": 0, "start": "2018-12-20 10:08:50.615333", "stderr": "", "stderr_lines": [], "stdout": "Healthy", "stdout_lines": ["Healthy"]}

TASK [Wait for volumes to be Rebuilding state] *********************************
task path: /percona/chaos/cstor_pool_scale/test.yml:137
skipping: [127.0.0.1] => {"changed": false, "skip_reason": "Conditional result was False"}

TASK [Recording the total allocated space in cvr] ******************************
task path: /percona/chaos/cstor_pool_scale/test.yml:146
changed: [127.0.0.1] => {"changed": true, "cmd": "kubectl get cvr pvc-aedf9cbb-0434-11e9-9649-0ac72a660b58-cstor-sparse-pool-6ect -n openebs -o jsonpath='{.status.capacity.totalAllocated}'", "delta": "0:00:00.379180", "end": "2018-12-20 10:08:51.568260", "rc": 0, "start": "2018-12-20 10:08:51.189080", "stderr": "", "stderr_lines": [], "stdout": "42.9M", "stdout_lines": ["42.9M"]}

TASK [Compareing the allocated space] ******************************************
task path: /percona/chaos/cstor_pool_scale/test.yml:150
changed: [127.0.0.1] => {"changed": true, "cmd": "kubectl get cvr pvc-aedf9cbb-0434-11e9-9649-0ac72a660b58-cstor-sparse-pool-4csk -n openebs -o jsonpath='{.status.capacity.totalAllocated}'", "delta": "0:00:00.381231", "end": "2018-12-20 10:08:52.103620", "failed_when_result": false, "rc": 0, "start": "2018-12-20 10:08:51.722389", "stderr": "", "stderr_lines": [], "stdout": "42.8M", "stdout_lines": ["42.8M"]}

TASK [set_fact] ****************************************************************
task path: /percona/chaos/cstor_pool_scale/test.yml:155
ok: [127.0.0.1] => {"ansible_facts": {"flag": "Pass"}, "changed": false}

TASK [include_tasks] ***********************************************************
task path: /percona/chaos/cstor_pool_scale/test.yml:166
included: /common/utils/update_litmus_result_resource.yml for 127.0.0.1

TASK [Generate the litmus result CR to reflect SOT (Start of Test)] ************
task path: /common/utils/update_litmus_result_resource.yml:3
skipping: [127.0.0.1] => {"changed": false, "skip_reason": "Conditional result was False"}

TASK [Analyze the cr yaml] *****************************************************
task path: /common/utils/update_litmus_result_resource.yml:14
skipping: [127.0.0.1] => {"changed": false, "skip_reason": "Conditional result was False"}

TASK [Apply the litmus result CR] **********************************************
task path: /common/utils/update_litmus_result_resource.yml:17
skipping: [127.0.0.1] => {"changed": false, "skip_reason": "Conditional result was False"}

TASK [Generate the litmus result CR to reflect EOT (End of Test)] **************
task path: /common/utils/update_litmus_result_resource.yml:27
changed: [127.0.0.1] => {"changed": true, "checksum": "f1a27710b9836ac8e98df83f87c49363cbc326c7", "dest": "./litmus-result.yaml", "gid": 0, "group": "root", "md5sum": "b883698002224f703c8176c7f1581590", "mode": "0644", "owner": "root", "size": 431, "src": "/root/.ansible/tmp/ansible-tmp-1545300532.41-69742881441698/source", "state": "file", "uid": 0}

TASK [Analyze the cr yaml] *****************************************************
task path: /common/utils/update_litmus_result_resource.yml:38
changed: [127.0.0.1] => {"changed": true, "cmd": "cat litmus-result.yaml", "delta": "0:00:00.300321", "end": "2018-12-20 10:08:53.671720", "rc": 0, "start": "2018-12-20 10:08:53.371399", "stderr": "", "stderr_lines": [], "stdout": "---\napiVersion: litmus.io/v1alpha1\nkind: LitmusResult\nmetadata:\n\n  # name of the litmus testcase\n  name: cstor-pool-scale \nspec:\n\n  # holds information on the testcase\n  testMetadata:\n    app:  \n    chaostype: cstor-pool-scale \n\n  # holds the state of testcase,  manually updated by json merge patch\n  # result is the useful value today, but anticipate phase use in future \n  testStatus: \n    phase: completed  \n    result: Pass ", "stdout_lines": ["---", "apiVersion: litmus.io/v1alpha1", "kind: LitmusResult", "metadata:", "", "  # name of the litmus testcase", "  name: cstor-pool-scale ", "spec:", "", "  # holds information on the testcase", "  testMetadata:", "    app:  ", "    chaostype: cstor-pool-scale ", "", "  # holds the state of testcase,  manually updated by json merge patch", "  # result is the useful value today, but anticipate phase use in future ", "  testStatus: ", "    phase: completed  ", "    result: Pass "]}

TASK [Apply the litmus result CR] **********************************************
task path: /common/utils/update_litmus_result_resource.yml:41
changed: [127.0.0.1] => {"changed": true, "cmd": "kubectl apply -f litmus-result.yaml", "delta": "0:00:00.511016", "end": "2018-12-20 10:08:54.336268", "failed_when_result": false, "rc": 0, "start": "2018-12-20 10:08:53.825252", "stderr": "", "stderr_lines": [], "stdout": "litmusresult.litmus.io/cstor-pool-scale configured", "stdout_lines": ["litmusresult.litmus.io/cstor-pool-scale configured"]}

TASK [Deleting test specific data from mysql database] *************************
task path: /percona/chaos/cstor_pool_scale/test.yml:171
included: /common/utils/mysql_data_persistence.yml for 127.0.0.1

TASK [Create some test data in the mysql database] *****************************
task path: /common/utils/mysql_data_persistence.yml:4
skipping: [127.0.0.1] => (item=mysql -uroot -pk8sDem0 -e 'create database tdb;')  => {"changed": false, "item": "mysql -uroot -pk8sDem0 -e 'create database tdb;'", "skip_reason": "Conditional result was False"}
skipping: [127.0.0.1] => (item=mysql -uroot -pk8sDem0 -e 'create table ttbl (Data VARCHAR(20));' tdb)  => {"changed": false, "item": "mysql -uroot -pk8sDem0 -e 'create table ttbl (Data VARCHAR(20));' tdb", "skip_reason": "Conditional result was False"}
skipping: [127.0.0.1] => (item=mysql -uroot -pk8sDem0 -e 'insert into ttbl (Data) VALUES ("tdata");' tdb)  => {"changed": false, "item": "mysql -uroot -pk8sDem0 -e 'insert into ttbl (Data) VALUES (\"tdata\");' tdb", "skip_reason": "Conditional result was False"}

TASK [Checking for the Corrupted tables] ***************************************
task path: /common/utils/mysql_data_persistence.yml:21
skipping: [127.0.0.1] => {"changed": false, "skip_reason": "Conditional result was False"}

TASK [Verify mysql data persistence] *******************************************
task path: /common/utils/mysql_data_persistence.yml:30
skipping: [127.0.0.1] => {"changed": false, "skip_reason": "Conditional result was False"}

TASK [Delete/drop MySQL database] **********************************************
task path: /common/utils/mysql_data_persistence.yml:43
changed: [127.0.0.1] => {"changed": true, "cmd": "kubectl exec percona-cb697f8b4-n4dgb -n app-percona-ns -- mysql -uroot -pk8sDem0 -e 'drop database tdb';", "delta": "0:00:01.270733", "end": "2018-12-20 10:08:55.997121", "rc": 0, "start": "2018-12-20 10:08:54.726388", "stderr": "mysql: [Warning] Using a password on the command line interface can be insecure.", "stderr_lines": ["mysql: [Warning] Using a password on the command line interface can be insecure."], "stdout": "", "stdout_lines": []}

TASK [Verify successful db delete] *********************************************
task path: /common/utils/mysql_data_persistence.yml:51
changed: [127.0.0.1] => {"changed": true, "cmd": "kubectl exec percona-cb697f8b4-n4dgb -n app-percona-ns -- mysql -uroot -pk8sDem0 -e 'show databases';", "delta": "0:00:00.453483", "end": "2018-12-20 10:08:56.642157", "failed_when_result": false, "rc": 0, "start": "2018-12-20 10:08:56.188674", "stderr": "mysql: [Warning] Using a password on the command line interface can be insecure.", "stderr_lines": ["mysql: [Warning] Using a password on the command line interface can be insecure."], "stdout": "Database\ninformation_schema\nmysql\nperformance_schema\nsys", "stdout_lines": ["Database", "information_schema", "mysql", "performance_schema", "sys"]}
META: ran handlers
META: ran handlers

PLAY RECAP *********************************************************************
127.0.0.1                  : ok=32   changed=25   unreachable=0    failed=0 
```

### CODE REVIEW

File: apps/percona/chaos/cstor_pool_scale/test.yml [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+
+            - name: Record test instance/run ID
+              set_fact:
+                run_id: "{{ lookup('env','RUN_ID') }}"
+
+            - name: Construct testname appended with runID
+              set_fact:
+                test_name: "{{ test_name }}-{{ run_id }}"
+
+          when: lookup('env','RUN_ID')
```
Comment (nsathyaseelan) [inline apps/percona/chaos/cstor_pool_scale/test.yml]: @shashank855  Can you use this util `/common/utils/create_testname.yml` to create test name.

File: apps/percona/chaos/cstor_pool_scale/test.yml [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+
+        - name: Deleting test specific data from mysql database
+          include_tasks: "/common/utils/mysql_data_persistence.yml"
+          vars:
+            status: 'DELETE'
+            ns: "{{ namespace }}"
+            pod_name: "{{ app_pod_name.stdout }}"
+            dbuser: 'root'
+            dbpassword: 'k8sDem0'
+            dbname: 'tdb'
```
Comment (nsathyaseelan) [inline apps/percona/chaos/cstor_pool_scale/test.yml]: @shashank855 create unique string and pass as a dbname.

File: apps/percona/chaos/cstor_pool_scale/test.yml [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+
+        - name: Create some test data in the mysql database
+          include_tasks: "/common/utils/mysql_data_persistence.yml"
+          vars:
+            status: 'LOAD'
+            ns: "{{ namespace }}"
+            pod_name: "{{ app_pod_name.stdout }}"
+            dbuser: 'root'
+            dbpassword: 'k8sDem0'
+            dbname: 'tdb'
```
Comment (nsathyaseelan) [inline apps/percona/chaos/cstor_pool_scale/test.yml]: @shashank855 create unique string and pass as a dbname.

File: apps/percona/chaos/cstor_pool_scale/test.yml (hunk 1 above)
Comment (shashank855) [reply apps/percona/chaos/cstor_pool_scale/test.yml]: @nsathyaseelan I have made the changes !!

File: apps/percona/chaos/cstor_pool_scale/test.yml (hunk 3 above)
Comment (shashank855) [reply apps/percona/chaos/cstor_pool_scale/test.yml]: @nsathyaseelan I have made the changes !!

**Pull request merged**